sample-app/src/main/java/org/smartregister/pnc/sample/config/PncRegisterQueryProvider.java

```diff
@@ -33,7 +33,7 @@ public String getObjectIdsQuery(@Nullable String filters, @Nullable String mainC
     public String[] countExecuteQueries(@Nullable String filters, @Nullable String mainCondition) {
         SmartRegisterQueryBuilder sqb = new SmartRegisterQueryBuilder();
         return new String[] {
-                sqb.countQueryFts("ec_client", null, "register_type = 'pnc'", filters)
+                sqb.countQueryFts("ec_client", null, null, filters)
```

Comment:
update this to use table specified in `PncMetadata`

Reply:
updated.